index.js

```diff
@@ -152,6 +164,18 @@ function asReqValue (req) {
   }
 }
 
+function buildNullLogger (levels) {
+  var logger = {}
+
+  var noop = function () { }
+
+  for (let i = 0; i < levels.length; i++) {
+    logger[levels[i]] = noop
+  }
```

Comment:
instead of building a null logger every time, you can use https://www.npmjs.com/package/abstract-logging.

Reply:
Ah, I see now.  I was trying to keep `hapi-pino` in that golden single external dep happy place a while longer. I can pull that in if you'd prefer.

Reply:
Add the dependency ;)

Reply:
Done.